app/controllers/student_identifier_types_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+class StudentIdentifierTypesController < ApplicationController
+  include OrganizationAuthorization
+
+  before_action :check_flipper
```

Comment:
Maybe instead of `:check_flipper` we can do something like `before_action :ensure_flipper_is_enabled`

/cc @johndbritton thoughts here?